### PR TITLE
elastic: bump to 6.5.2

### PR DESCRIPTION
**Pull request by hydrapolic**
Thanks :)

Tested on ~amd64.

### CODE REVIEW

File: app-misc/elasticsearch/elasticsearch-6.5.2.ebuild [hunk 1]
```diff
@@ -0,0 +1,91 @@
+# Copyright 1999-2018 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
```
Comment (Zlogene) [inline app-misc/elasticsearch/elasticsearch-6.5.2.ebuild]: why not 7?

**Comment by hydrapolic** [comment]
IIRC, some of the inherited eclass wasn't compatible with it.

File: app-misc/elasticsearch/elasticsearch-6.5.2.ebuild (hunk 1 above)
Comment (hydrapolic) [reply app-misc/elasticsearch/elasticsearch-6.5.2.ebuild]: IIRC, some of the inherited eclass wasn't compatible with it. But i'll retest tomorrow.

**Comment by Zlogene** [comment]
https://github.com/gentoo/gentoo/commit/2c5502e2948c1589de6e6810b5769034f35b2613 .. https://github.com/gentoo/gentoo/commit/d12f25e051efb10806dd390e78cc186a7248eb6b

**Pull request closed without merging**